nntrainer/dataset/raw_file_data_producer.cpp

```diff
@@ -46,7 +46,7 @@ RawFileDataProducer::size(const std::vector<TensorDim> &input_dims,
                                 sample_size, size_accumulator);
 
   auto path_prop = std::get<props::FilePath>(*raw_file_props);
-  auto file_size = path_prop.file_size();
+  // auto file_size = path_prop.file_size();
```

Comment:
Commented out codes are destined to confuse you weeks later.
Unless the commented codes are usage examples or suggestions for users, remove it.

If you are commenting out codes for future potential usage, make another commit that removes this line only and write a good title and description in the commit so that you can do ```git revert``` later.

Reply:
On a side note, shouldn't L61 be `return file_size / (sample_size * RawFileDataProducer::pixel_size);`?

Reply:
Oh you are right @kparichay. Thanks for the letting me know that :)